marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/FinishHubDeploymentCommand.java

```diff
@@ -0,0 +1,23 @@
+package com.marklogic.hub.deploy.commands;
+
+import com.marklogic.appdeployer.command.AbstractCommand;
+import com.marklogic.appdeployer.command.CommandContext;
+import com.marklogic.appdeployer.command.SortOrderConstants;
+import com.marklogic.hub.HubConfig;
+import com.marklogic.hub.dataservices.SystemService;
+import com.marklogic.hub.impl.HubConfigImpl;
+
+public class FinishHubDeploymentCommand extends AbstractCommand {
+
+    private HubConfigImpl hubConfig;
+
+    public FinishHubDeploymentCommand(HubConfig hubConfig) {
+        this.hubConfig = (HubConfigImpl) hubConfig;
+        setExecuteSortOrder(SortOrderConstants.DEPLOY_TRIGGERS + 1);
```

Comment:
I think this should be max integer to ensure this runs after everything else is done, which is what the name implies.